unabridged-works-volume-1/original/07-axe-to-the-root.adoc

```diff
@@ -352,6 +353,10 @@ And then consider where it is shut out, what it cannot do, what change it cannot
 what it cannot conform to: that so the true distinction may be let into the mind,
 and not a foundation laid of so great a mistake in a matter of so great concernment.
 
+[.numbered-block]
```

Comment:
Linter caught this, but it should be `[.numbered-group]`.  I'm curious though, did the editor show you this lint error? It should have...  Can you let me know if it didn't when you go back to fix this line? You should see a yellow lint warning box on the line number area.

Reply:
Yes, it did show me, I just didn't scroll through the whole thing at the end to catch any of those. Sometimes I forget to! Idk why I call them blocks!! :)